extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/recording/RecordableBootstrap.java

```diff
@@ -46,9 +45,7 @@ public RecordableBootstrap(BootstrapServiceRegistry bootstrapServiceRegistry,
     }
 
     private static Map initialProperties() {
-        HashMap map = new HashMap();
-        map.putAll(QuarkusEnvironment.getInitialProperties());
-        return map;
+        return new HashMap();
```

Comment:
`Collections.emptyMap()` wouldn't work here?

Reply:
Or better, remove the settings attribute completely?

Reply:
> `Collections.emptyMap()` wouldn't work here?

It needs to be mutable, as this is the instance to which we contribute other properties later.

> Or better, remove the settings attribute completely?

I like your suggestion and gave it a shot.. unfortunatley it got messy because of the weird requirement of having to extend `StandardServiceRegistryBuilder`, so let's keep this PR simple and focused - there's certainly more we can improve here but I'd try to get rid of the inheritance requirement first.

Pushing an update which removes this `initialProperties` method :+1: